frontend/src/features/permits/pages/TermOversize/components/form/PermitDetails.tsx

```diff
@@ -47,6 +48,12 @@ export const PermitDetails = ({
   // Permit expiry date === Permit start date + Permit duration - 1
   const expiryDate = dayjs(startDate).add(duration - 1, "day");
 
+  // handle leap year - if the given year is a leap year, and our
+  // expiry date is any month other then January, add 1 day back
+  if (expiryDate.isLeapYear() && expiryDate.month() > DATE_MONTHS.JAN) {
```

Comment:
Is it worth it to reduce the statement to this? The only advantage if this works is that we can do away with the enum.
```suggestion
  if (expiryDate.isLeapYear() && expiryDate.isAfter(expiryDate.year() + "01-31", 'month') ) {
```